src/RadixDlt.NetworkGateway.GatewayApiSdk/Model/PaginableEntityCoursor.cs

```diff
@@ -67,14 +67,14 @@
 namespace RadixDlt.NetworkGateway.GatewayApiSdk.Model;
 
 [DataContract]
-public sealed record EntityFungiblesCursor(int? Offset)
+public sealed record PaginableEntityCoursor(int? Offset)
```

Comment:
I'd change the name to `OffsetCursor`. There's no guarantee that every entity-related pagination will be offset based.